src/lib/epics/worker.ts

```diff
@@ -0,0 +1,31 @@
+import * as debug from "debug";
+import { Epic } from "redux-observable";
+import { map, concatMap } from "rxjs/operators";
+import { Action } from "redux-actions";
+
+import { IManifest } from "../types";
+import { remove } from "../actions";
+import { ADD, RESTORE } from "../constants/actions";
+import { send } from "../utils";
+
+const NAMESPACE = "@SL/http-queue:epics:send";
+
+const dbg = debug(NAMESPACE);
+
+export const sendMessagesEpic: Epic<Action<any>, Action<number>> = action$ =>
```

Comment:
The typing of Epic unfortunately makes it not possible to type it directly since the input-action is used for extending the output-action. This results in number not being exentable from `IManifest`. For the scope of this lib i'd leave it as is and not generate a new IEpic type as done in our other projects